app/controllers/albums.js

```diff
@@ -0,0 +1,23 @@
+const logger = require('../logger'),
+  bcrypt = require('bcryptjs'),
+  sessionManager = require('../services/sessionManager'),
+  User = require('../models').users,
+  userMw = require('../middlewares/users'),
+  errors = require('../errors'),
+  fetch = require('node-fetch');
+
+const URL = 'https://jsonplaceholder.typicode.com';
+
+exports.getAll = (request, response, next) => {
+  if (!request.userLogged)
+    return response.status(400).send(errors.authenticationError(['You must be logged to see all albums']));
+  fetch(`${URL}/albums`)
```

Comment:
We have a convention on Node projects where we create a `app/services/` folder, and we use it to store all methods involved on interacting with external services. In this particular case, you should create a file called `app/services/albums.js` and define there a method that calls this `.fetch()` and returns then response; and then you simply call that method from here.